packages/pug/test/pug.test.js

```diff
@@ -979,12 +979,20 @@ describe('pug', function(){
       assert(actual === expected);
     });
     it('should support .pug.compileClient(str, options)', function () {
-      var src = '.bar= self.foo'
+      var src = '.bar= self.foo';
       var fn = pug.compileClient(src, {self: true});
       fn = Function('pug', fn.toString() + '\nreturn template;')(pug.runtime);
       var actual = fn({foo: 'baz'});
       assert(actual === '<div class="bar">baz</div>');
     });
+    it('should support module syntax .pug.compileClient(str, options)', function () {
+      var src = '.bar= self.foo';
+      var fn = pug.compileClient(src, {self: true, module: true, inlineRuntimeFunctions: false});
```

Comment:
Ok, I added the second test. I have doubts about how to implement this `expect(fn).toMatchSnapshot();` because in the `.compileClient()` Test I didn't see how this can be used.